src/views/NutritionView.vue

```diff
@@ -271,10 +274,10 @@ export default {
       ); // The status of the response is not displayed so no need to wait the response
       }
     },
-    resetProductData() {
-      const data = {};
+    async resetProductData() {
+      const data = {};      
```

Comment:
I assume you do not need this function to be async

```suggestion
    resetProductData() {
      const data = {};
```